src/cmd_line/commands/normal.ts

```diff
@@ -0,0 +1,41 @@
+import { Parser, all, seq, whitespace } from 'parsimmon';
+import { VimState } from '../../state/vimState';
+import { ExCommand } from '../../vimscript/exCommand';
+import { LineRange } from '../../vimscript/lineRange';
+
+export class NormalCommand extends ExCommand {
+  // TODO: support to parse `:normal!`
+  public static readonly argParser: Parser<NormalCommand> = seq(whitespace, all).map(
+    ([_, keystroke]) => new NormalCommand(keystroke),
```

Comment:
This would be a bit nicer as:
```ts
whitespace.then(all).map(...);
```

Reply:
Fixed on [3149e8c](https://github.com/VSCodeVim/Vim/pull/8896/commits/3149e8c067bfea606c8280fd3c17b6aac18a90ca)